### ISSUE TITLE
A11y_Azure Tools Storage Explorer_Settings_Application_ScreenReader:Narrator doesn't read heading "Proxy" and the first paragraph present under the heading "Proxy" in scan mode.

### ISSUE THREAD

**Issue by ManishaC1** [microsoft/AzureStorageExplorer#5725]
**GitHub Tags:**
#A11yMAS; #A11yTCS; #Win11; #DesktopApp; #A11ySev2; #Azure Tools Storage Explorer; #MAS1.3.1; #ScreenReader; #Narrator; #FTP; #Benchmark;

## Actual results:
Narrator is not reading the heading "Proxy" and the first paragraph present under the heading-"A list of hostname.... app proxy settings." in scan mode.

**Similar issues are seen in the following.**

- Transfers

1. Launch Storage Explorer.
2. Navigate to Setting menu on left pane and click on it.
3. Navigate to "Transfers" on right pane and  click on it.
4. "Transfers" screen gets opened.
5. Turn on the Narrator in scan on mode.
6. Navigate to "Maximum transfer Rate" heading and its text. Verify they are MAS compliant.

## Expected results:
Narrator should read the heading and the paragraph.

## Environmental Details:
Application: Microsoft Azure Storage Explore
version: 1.24.0

## OS Details
Microsoft Windows 11 Enterprise
version 21H2 OS Build 22000.613

## Repro Steps:

1. Launch Storage Explorer
2. Navigate to Setting menu on left pane and click on it.
3. Navigate to Application on right pane and  click on it.
4. "Application" screen gets opened.
5. Turn on the Narrator in scan on mode.
6.  Navigate to "Proxy" heading and verify it is MAS compliant.

## User Impact:
User might get confused if radio buttons on focus is being read as "heading level 1".

"[Check out Accessibility Insights](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0)! - Identify accessibility bugs before check-in and make bug fixing faster and easier."

## Attachments
https://user-images.githubusercontent.com/105198219/168302518-2c0a21ad-dbdb-4823-b9f4-1bff47761e56.mp4

https://user-images.githubusercontent.com/105198219/168305094-f3dd5b91-34eb-41a3-8f7f-b5eaa10d0703.mp4

**Comment by ManishaC1** [comment]
Closing this bug as it is not reproducing while using NVDA.
